src/Features/Core/Portable/GenerateConstructorFromMembers/AbstractGenerateConstructorFromMembersCodeRefactoringProvider.cs

```diff
@@ -202,6 +202,12 @@ public override Task ComputeRefactoringsAsync(CodeRefactoringContext context)
                 return null;
             }
 
+            // We shouldn't offer a refactoring if the compilation is invalid.
```

Comment:
this comment isn't valid.  The compilation may be 100% valid, it may just not have this type in it :)